### PR TITLE
BNtoConv: volumetric support

### PR DESCRIPTION
**Pull request by mys007**
The original code actually already supports volumetric batchnorm integration, so I just allowed it.
Btw: I think you forgot to add `utils` to `init.lua`?

### CODE REVIEW

File: utils.lua [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 -- and BatchNormalization to Linear
 local function BNtoConv(net)
   for i,v in ipairs(net.modules) do
     if v.modules then
       BNtoConv(v)
     else
-      if (torch.typename(v):find'nn.SpatialBatchNormalization') and net:get(i-1) and
-        (torch.typename(net:get(i-1)):find'SpatialConvolution') then
+      if net:get(i-1) and 
+        ((torch.typename(v):find'nn.SpatialBatchNormalization' and torch.typename(net:get(i-1)):find'SpatialConvolution') or
```
Comment (szagoruyko) [inline utils.lua]: can you split it in 2 lines?

**Comment by szagoruyko** [comment]
about `utils` in `init.lua`, I prefer to use it like this:
```lua
local utils = require 'inn.utils'
utils.foldBatchNorm(net)
```
without registering in `inn` table

**Comment by mys007** [comment]
Here you go.

**Comment by szagoruyko** [comment]
thanks

**Pull request merged**